src/main/scala/fs2/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -491,8 +492,10 @@ private[kafka] object KafkaConsumerActor {
       deferred.complete((chunk, FetchCompletedReason.FetchedRecords))
   }
 
+  type StreamId = Int
```

Comment:
There are a few places in the pull request where we don't use this alias, but just say `Int` instead.

Reply:
This type alias is only here for documentation purpose:
```
fetches: Map[TopicPartition, Map[StreamId, FetchRequest[F, K, V]]],
```
instead of
```
fetches: Map[TopicPartition, Map[Int, FetchRequest[F, K, V]]],
```
Everywhere else, we have a named parameter: `streamId: StreamId`. I you'd want to be type-safe you'd create another type wrapping Int, but since this is an internal API I fine with this (saving some memory).

Reply:
I meant that we could say `StreamId` everywhere where we say `Int`, including e.g. `streamId: StreamId`, `Ref[F, StreamId]`, etc.